dashboard.php

```diff
@@ -5,155 +5,67 @@
-                        <div class="job-edit">
-                            <a href="#">View Submissions</a>
-                            <a href="#">Edit</a>
-                        </div>
-                    </div>
-                </li>
-                <li class="job-card">
-                    <div class="job-primary">
-                        <h2 class="job-title"><a href="#">Front End Developer</a></h2>
-                        <div class="job-meta">
-                            <a class="meta-company" href="#">Company Awesome Ltd.</a>
-                            <span class="meta-date">Posted 14 days ago</span>
-                        </div>
-                        <div class="job-details">
-                            <span class="job-location">The Hague (The Netherlands)</span>
-                            <span class="job-type">Contract staff</span>
-                        </div>
-                    </div>
-                    <div class="job-secondary">
-                        <div class="job-edit">
-                            <a href="#">View Submissions</a>
-                            <a href="#">Edit</a>
-                        </div>
-                    </div>
-                </li>
-                <li class="job-card">
-                    <div class="job-primary">
-                        <h2 class="job-title"><a href="#">Front End Developer</a></h2>
-                        <div class="job-meta">
-                            <a class="meta-company" href="#">Company Awesome Ltd.</a>
-                            <span class="meta-date">Posted 14 days ago</span>
-                        </div>
-                        <div class="job-details">
-                            <span class="job-location">The Hague (The Netherlands)</span>
-                            <span class="job-type">Contract staff</span>
-                        </div>
-                    </div>
-                    <div class="job-secondary">
-                        <div class="job-edit">
-                            <a href="#">View Submissions</a>
-                            <a href="#">Edit</a>
-                        </div>
-                    </div>
-                </li>
-            </ul>
-            <div class="jobs-pagination-wrapper">
-                <div class="nav-links">
-                    <a class="page-numbers current">1</a>
-                    <a class="page-numbers">2</a>
-                    <a class="page-numbers">3</a>
-                    <a class="page-numbers">4</a>
-                    <a class="page-numbers">5</a>
                 </div>
             </div>
         </div>
-    </section>
+		<?php require_once( $_SERVER['DOCUMENT_ROOT'] . '/includes/theme-compat/jobs-listing.php' );
+
+		$s_parameter = '';
+		if ( ! empty( $search ) ) {
+			$s_parameter = '?search=' . $search;
+		}
+		$page_url    = 'dashboard.php' . $s_parameter;
+		$active_jobs = active_jobs();
+		pagination( $active_jobs, $page_url ); ?>
```

Comment:
Your pagination is wrong. It will show the wrong number when you have search or filter applied.
`active_jobs` always returns the total number of all active jobs.